packages/connectkit/src/hooks/usePaymentState.ts

```diff
@@ -61,6 +62,8 @@ export interface PayParams {
   preferredChains?: number[];
   /** Preferred tokens. These appear first in the token list. */
   preferredTokens?: { chain: number; address: Address }[];
+  /** Developer metadata. E.g. correlation ID. */
+  metadata?: DaimoPayUserMetadata;
```

Comment:
confirming that we don't expose our previous `metadata` anywhere... fortunately, it looks like we don't